libs/neo4j/tests/integration_tests/chains/test_graph_database.py

```diff
@@ -85,18 +46,14 @@ def test_cypher_generating_run() -> None:
     assert output == expected_output
 
 
-def test_cypher_top_k() -> None:
+@pytest.mark.usefixtures("clear_neo4j_database")
+def test_cypher_top_k(neo4j_credentials: Dict[str, str]) -> None:
     """Test top_k parameter correctly limits the number of results in the context."""
-    url = os.environ.get("NEO4J_URI", "bolt://localhost:7687")
-    username = os.environ.get("NEO4J_USERNAME", "neo4j")
-    password = os.environ.get("NEO4J_PASSWORD", "pleaseletmein")
-
     TOP_K = 1
-
     graph = Neo4jGraph(
-        url=url,
-        username=username,
-        password=password,
+        url=neo4j_credentials["url"],
```

Comment:
Small detail, but I think:
```suggestion
  graph = Neo4jGraph(**neo4j_credentials)
```
could save us 0.1 seconds in the future :-P (OK we've invented copy/paste, I know ^^)

Reply:
I'm always up for saving some time however small! Updated in [15dd4a1](https://github.com/langchain-ai/langchain-neo4j/pull/50/commits/15dd4a14879ccae7cab0c8b2b76471da1fa87fac)